config/initializers/honeycomb.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../../app/lib/honeycomb/noise_cancelling_sampler"
```

Comment:
Without this, I got a `NameError` from the Sidekiq workers when starting the app locally with `overmind start -f Procfile.dev`.

Reply:
Another use case for [`Rails.application.config.to_prepare`](https://guides.rubyonrails.org/v7.0/autoloading_and_reloading_constants.html#use-case-1-during-boot-load-reloadable-code).